README.md

```diff
@@ -0,0 +1,13 @@
+META-PELUX-RPI-BSP
+====================
+Yocto layer adding configurations required when building PELUX towards raspberry pi.
+
+Maintained at https://github.com/pelagicore/meta-pelux-bsp-rpi
```

Comment:
There should be a capital P in Pelagicore: https://github.com/Pelagicore/meta-pelux-bsp-rpi